addons/rose/addon/components/rose/form/input/index.stories.mdx

```diff
@@ -13,7 +13,7 @@ Renders a simple form input.
 <Rose::Form::Input
   @type="email"
   @name="email"
-  @value="info@example.com"
```

Comment:
Can this be changed back to what it originally was?

Reply:
done!